client.go

```diff
@@ -157,6 +157,12 @@ func (c *Client) getCAS(key string, ocas uint64) (val string, flags uint32, cas
 	}
 
 	err = c.perform(m)
+	if c.config.Compression.unzip != nil && err == nil {
```

Comment:
Probably best to check `config.Compression != nil` too for being as safe as possible

Reply:
ok, done